src/components/hero.jsx

```diff
@@ -101,8 +101,8 @@ const Hero = () => {
         </div>
       </section>
 
-      <section className=" justify-center gap-10 mt-10 items-center flex flex-col lg:flex-row">
-        <div className="border-2 p-4 rounded-lg border-purple md:rotate-3 relative md:mt-0 dark:border-deeppurple">
+      <section className=" justify-center lg:gap-10 mt-10 items-center flex flex-col lg:flex-row">
+        <div className="border-2 p-4 rounded-lg border-purple lg:rotate-3 hover:lg:rotate-0 hover:lg:ease-in-out hover:lg:duration-300 relative md:mt-0 dark:border-deeppurple">
```

Comment:
Looked at this animation for a bit again. On enter, it eases in just fine with a nice duration. But on exit of hover, it immediately just springs back to its normal position, and that does not feel right.